### DISCUSSION TITLE
Issue upgrading to 0.7

### DISCUSSION THREAD

**Discussion by johnpaulcas** [wevm/wagmi#1276] [General]
Im having logs error after updating wagmi version to 0.7, I never used isDataEqual on my code, any way to resolve this error?

<img width="482" alt="Screen Shot 2022-11-10 at 8 00 19 AM" src="https://user-images.githubusercontent.com/44362950/200968004-ca1aab01-e09c-4c5a-829e-c36a4289b5e6.png">

**Comment by tmm** [comment]
What 0.7 version? This was fixed in 0.7.15.

**Reply by johnpaulcas** [reply]
ohw I see, Im using 0.7.13, btw may I ask if its possible to set the connected wallet like metamask as default provider so we don't get or atleast prevent the 429 issue rate-limit on infura or alchemy

**Reply by tmm** [reply]
It's not currently possible, but we are looking into it.

**Reply by johnpaulcas** [reply]
okay, guess thats one big feature that missing to this project, guess it will solve a lot of issue like rate-limiting, btw already fixed the issue upgrading my to 0.7.15, thanks @tmm
